assets/components/molecules/card-file.html.swig

```diff
@@ -0,0 +1,74 @@
+---
+title: Card File
+name: card-file
+collection: molecules
+variants:
+  grp: Group files
+notes: |
+  This is an implementation of the card component.
+---
+<div class="card card-no-border card-no-padding card-no-shadow">
+  <div class="card-body">
```

Comment:
Why do we need to put it in a card component? It's using basically nothing from it 🤔 

I tested by removing the `.card` and `.card-body` wrappers and nothing changes. Delete them!